Gopkg.toml

```diff
@@ -38,14 +38,12 @@ required = [
   name = "github.com/docker/distribution"
   branch = "master"
 
-# TODO: switch to 1.10 and newer client-go
-
-# client-go release-6.0 corresponds to k8s release-1.9 branch. See
-# https://github.com/kubernetes/client-go/tree/release-6.0#compatibility-matrix
+# client-go release-6.0 corresponds to k8s release-1.9 branch and release-7.0 is 1.10. See
```

Comment:
Can you remove the first sentence here? The compatibility-matrix reference is sufficient.